src/libraries/Silk.Net.Windowing/Abstract/ISurface.cs

```diff
@@ -0,0 +1,162 @@
+﻿using Silk.NET.Core;
```

Comment:
All files need the .NET Foundation license header.

```cs
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
```

Reply:
yes, fixed that on my end